e2e/playwright/named-views.spec.ts

```diff
@@ -0,0 +1,292 @@
+    tomlString = tomlStringOverWriteNamedViewUuids(tomlString)
+
+    // Write the entire tomlString to a snapshot.
+    // There are many key/value pairs to check this is a safer match.
+    expect(tomlString).toMatchSnapshot('verify-named-view-gets-created')
+
+    // Delete a named view
+    await cmdBar.openCmdBar()
+    await cmdBar.chooseCommand('delete named view')
+    cmdBar.selectOption({ name: myNamedView2 })
+    await cmdBar.progressCmdBar(false)
+
+    // Read project.toml into memory again since we deleted a named view
+    tomlString = await fsp.readFile(tempProjectSettingsFilePath, 'utf-8')
+    // Rewrite the uuids in the named views to match snapshot otherwise they will be randomly generated from rust and break
+    tomlString = tomlStringOverWriteNamedViewUuids(tomlString)
+
+    // // Write the entire tomlString to a snapshot.
+    // // There are many key/value pairs to check this is a safer match.
+    expect(tomlString).toMatchSnapshot('verify-named-view-gets-deleted')
+  })
+  test('Verify named view gets loaded', async ({
+    cmdBar,
+    scene,
+    page,
+  }, testInfo) => {
+    const projectName = 'named-views'
+    const myNamedView = 'uuid1'
+
+    // Create project and go into the project
+    await createProject({ name: projectName, page })
+    await scene.waitForExecutionDone()
+
+    // Create a new named view
+    await cmdBar.openCmdBar()
+    await cmdBar.chooseCommand('create named view')
+    await cmdBar.argumentInput.fill(myNamedView)
+    await cmdBar.progressCmdBar(false)
+
+    // Generate file paths for project.toml
+    const projectDirName = testInfo.outputPath('electron-test-projects-dir')
+    const tempProjectSettingsFilePath = join(
+      projectDirName,
+      projectName,
+      PROJECT_SETTINGS_FILE_NAME
+    )
+
+    // Except the project.toml to be written to disk since a named view was created
+    await expect(async () => {
+      let exists = await fileExists(tempProjectSettingsFilePath)
+      expect(exists).toBe(true)
+    }).toPass()
+
+    // Read project.toml into memory
+    let tomlString = await fsp.readFile(tempProjectSettingsFilePath, 'utf-8')
+    // Rewrite the uuids in the named views to match snapshot otherwise they will be randomly generated from rust and break
+    tomlString = tomlStringOverWriteNamedViewUuids(tomlString)
+
+    // Write the entire tomlString to a snapshot.
+    // There are many key/value pairs to check this is a safer match.
+    expect(tomlString).toMatchSnapshot('verify-named-view-gets-created')
+
+    // Create a load a named view
+    await cmdBar.openCmdBar()
+    await cmdBar.chooseCommand('load named view')
+    await cmdBar.argumentInput.fill(myNamedView)
+    await cmdBar.progressCmdBar(false)
+
+    // Check the toast appeared
+    await expect(
+      page.getByText(`Named view ${myNamedView} loaded.`)
+    ).toBeVisible()
```

Comment:
This test doesn't verify the camera actually moves to the loaded view's position. Should it?